sections/sprint-1/experiment.tex

```diff
@@ -47,76 +47,116 @@ \subsection{Setup}
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=1\linewidth]{1tagdiagram.PNG}
+    \caption{A boxplot showing the dispersion of measured positions with 1 tag}
+    \label{fig:1tagdiagram}
+\end{figure}
+As can be seen on figure \ref{fig:1tagdiagram} some measurements can be quite far off the actual position. 
+On the x-axis you can see the actual position of the tag in mm compared to the playing field.
+On the y-axis is the amount in mm that the measurements were off the actual position.
+The boxplot shows the minimum and maximum values aswell as the distribution of the measurements according to what quartile it is in.
+The box is the measurements withing first and third quartile and the whiskers that are extended out from it is the measurements on outside of the these quartiles.
+Interestingly it is the positions closest to x-axis that is the most inaccurate.
+The measurement with the highest deviance was recorded when the tag was in position \texttt{(0,3000)} were the measured position was 1.3 meters off.
+But even then the average deviance of all the measurements from that position was only 36.5 cm.
+The highest average deviance of all the positions was 39.5 cm in position \texttt{(0,6000)}
+When running with one tag the average amount of measurements were 12.368 per second.
+ 
 
 \subsubsection{Precision with 3 tag}
 For this part of the experiment, the accuracy was tested with three tags. 
 The tags were positioned at three different x-coordinates. 
-140, 180 and 220 were chosen as the $x$ coordinates and the $y$ coordinate was moved up by 30 cm each time starting from 0 to 120.
+1400, 1800 and 2200 were chosen as the $x$ coordinates and the $y$ coordinate was moved up by 300 mm each time starting from 0 and going up to 1200.
+The average amount of updates from all the tags per second were 1.83, less than 10 percent of the amount of updates when only one tag was used.
+The amount of updates were very different between the different tags as the one with the highest update rate had an average of 3.4 per second and the lowest only 0.92.
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=1\linewidth]{3tagdiagram.PNG}
+    \caption{A boxplot showing the dispersion of measured positions with 3 tags}
+    \label{fig:3tagdiagram}
+\end{figure}
+On figure \ref{fig:3tagdiagram} all the measurements from the 3 tag experiment can be seen.
+The 3 last sets of measurements are from when all the tags were placed close to each other at the same point.
+This was done to see if they would affect the precision of each other.
+Based on the data it seems they might have a small effect on each other but not enough for it to be a problem.
+The biggest deviance in the test was only 46 cm off compared to 1.3 meters in the 1 tag test which seemed interesting, but could be because the measurements in the 3 tag experiment was made in the middle of the playing field compared to at the edge in the 1 tag experiment.
```

Comment:
```suggestion
The biggest deviance in the test was only 46 cm off compared to 1.3 meters in the 1 tag test which seemed interesting, but could be because the measurements in the 3 tag experiment were made in the middle of the playing field compared to at the edge in the 1 tag experiment.
```

Reply:
måske lige nævne hurtgit hvad problemet var når de var ved edgen. Var det pga metal der fuckede med det eller var det bare fordi det var svært at håndtere derude?